css/stylesDesktop.css

```diff
@@ -0,0 +1,313 @@
+  .contact-links.headline.social-media-icons {
+    display: none;
+  }
+
+  /* Footer */
+  .contact-email {
+    display: none;
+  }
+
+  /* Changed Elements */
+
+  /* Header Section */
+  .toolbar-container {
+    top: 0;
+    justify-content: center;
+  }
+
+  .toolbar.menu {
+    margin: 0;
+  }
+
+  .toolbar-menu-desktop {
+    margin-top: 14px;
+    display: inline-flex;
+    width: 255px;
+    justify-content: space-between;
+  }
+
+  .desktop-menu-item {
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .toolbar-menu-desktop li {
+    transition: all ease-in-out 0.5s;
+  }
+
+  .toolbar-menu-desktop li:hover {
+    border-bottom: 2px solid green;
+    transform: translateY(10%);
+  }
+
+  .headline.primary-text.flex-container-column {
+    margin-top: 262px;
+  }
+
+  .about-me.text {
+    margin-top: 168px;
+  }
+
+  .supporting-text {
+    width: 70vw;
+    max-width: 1024px;
+  }
+
+  /* Recents Works Section */
+
+  /* modal styles */
+  .recent-works-detailed {
+    max-width: 920px;
+    max-height: calc(1083px - 32px);
+    align-items: flex-start;
+  }
+
+  .popup-work-img {
+    width: 872px;
+    height: 616px;
+    margin: 78px 24px 0;
+  }
+
+  .popup-close-icon {
+    position: absolute;
+    right: calc((100vw - 1000px) / 2 + 24px);
+    top: 24px;
+    margin: 0 24px 0;
+  }
+
+  .popup-header {
+    max-width: 589px;
+    margin: 0 24px 0;
+  }
+
+  .popup-techs {
+    justify-content: flex-start;
+  }
+
+  .popup-tag {
+    margin-right: 8px;
+    margin-left: 24px;
+  }
+
+  .popup-p-font {
+    max-width: 90%;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+    text-align: left;
+    justify-content: space-around;
+    align-items: center;
+    margin: 0 24px 0;
+  }
+
+  .popup-btn-container {
+    position: absolute;
+    bottom: 249px;
+    right: calc((100vw - 1000px) / 2 + 24px + 24px);
+    width: auto;
+    display: inline-flex;
+  }
+
+  .popup-btn-font {
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    padding: 12px 16px 0 16px;
+    margin-right: 12px;
+  }
+
+  .mb-12 {
+    margin-bottom: 24px;
+  }
+
+  /* Recents Works Header */
+  .recent-works.intro-header {
+    justify-content: space-between;
+    width: auto;
+    height: auto;
+    margin-top: 30px;
+    margin-bottom: 52px;
+  }
+
+  .intro-header.primary-text > h2 {
+    margin-top: 0;
+    width: 100%;
+  }
+
+  .intro-header.indicator {
+    display: none;
+  }
+
+  /* Display Work section */
+  .card-work-container {
+    grid-template: repeat(2, 1fr) / repeat(3, 1fr);
+    width: 81vw;
+    max-width: 1200px;
+    justify-items: center;
+  }
+
+  .card-work {
+    align-items: center;
+    justify-content: space-between;
+    min-height: 474px;
+    width: 91%;
+    border-radius: 8px;
+    border: 1px solid #d0d9d4;
+    background-color: #ebf0ee;
+    margin-bottom: 24px;
+  }
```

Comment:
Your work cards are not matching the Figma template of your choosing. Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/228822474-fdcf1f09-d539-48ae-8858-980de06b8fe0.png) | ![image](https://user-images.githubusercontent.com/33268927/228822343-ac5f2872-c411-437a-bca5-28093e6b088b.png) |